src/NuGet.Clients/NuGet.PackageManagement.UI/PackageLoadContext.cs

```diff
@@ -18,6 +19,7 @@ namespace NuGet.PackageManagement.UI
     internal class PackageLoadContext
     {
         private readonly Task<PackageCollection> _installedPackagesTask;
+        private readonly JoinableTask<(PackageCollection, PackageCollection)> _allPackagesTask;
```

Comment:
value tuples allow us to give each value a name, so it's much more clear what it actually is. I'm just guessing, since I haven't yet seen the method implementation, but something like this:

```suggestion
        private readonly JoinableTask<(PackageCollection directPackages, PackageCollection transitivePackages)> _allPackagesTask;
```

Reply:
Fixed